### PR TITLE
Update to editor and plugins with updated dependencies

### PR DESCRIPTION
**Pull request by piemonkey**
### Overview
Includes some dependency updates but only those required to support the latest editor and plugins.

##### connected issues and PRs:
https://github.com/lblod/ember-rdfa-editor/pull/1204
https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/448

### Setup
N/A

### How to test/reproduce
Everything should work as before.

### Challenges/uncertainties
There is an issue with the agendapoints editor as [it imports `besluitNodes`](https://github.com/lblod/frontend-gelinkt-notuleren/blob/f48e125d54d60ebce90ca748eb8f73b1faea429b/app/controllers/agendapoints/edit.js#L67) from plugins which [has been removed](https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/449). I think it makes sense to make any changes for this in a separate PR.

### Checks PR readiness
- [ ] UI: works on smaller screen sizes
- [ ] UI: feedback for any loading/error states
- [ ] Check cancel/go-back flows
- [ ] Check database state correct when deleting/updating (especially regarding relationships)
- [ ] changelog
- [ ] npm lint
- [ ] no new deprecations

### CODE REVIEW

**Review by elpoelma — COMMENTED** [review]
Not really a problem with this PR, but there still seem to be some issues with the latest release of the editor-plugins concerning the `decision` nodes etc.

**Comment by piemonkey** [comment]
> Not really a problem with this PR, but there still seem to be some issues with the latest release of the editor-plugins concerning the `decision` nodes etc.

You mean other issues than the one I mentioned in the description?

> There is an issue with the agendapoints editor as [it imports besluitNodes](https://github.com/lblod/frontend-gelinkt-notuleren/blob/f48e125d54d60ebce90ca748eb8f73b1faea429b/app/controllers/agendapoints/edit.js#L67) from plugins which https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/449. I think it makes sense to make any changes for this in a separate PR.

**Comment by elpoelma** [comment]
> > Not really a problem with this PR, but there still seem to be some issues with the latest release of the editor-plugins concerning the `decision` nodes etc.
> 
> You mean other issues than the one I mentioned in the description?
> 
> > There is an issue with the agendapoints editor as [it imports besluitNodes](https://github.com/lblod/frontend-gelinkt-notuleren/blob/f48e125d54d60ebce90ca748eb8f73b1faea429b/app/controllers/agendapoints/edit.js#L67) from plugins which [lblod/ember-rdfa-editor-lblod-plugins#449](https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/449). I think it makes sense to make any changes for this in a separate PR.

Ah wow, totally glanced over that.

File: package.json [hunk 1]
```diff
@@ -43,11 +43,11 @@
...
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "@lblod/ember-acmidm-login": "^2.0.0-beta.1",
+    "@lblod/ember-acmidm-login": "^2.1.0",
     "@lblod/ember-environment-banner": "^0.5.0",
-    "@lblod/ember-mock-login": "0.7.0",
-    "@lblod/ember-rdfa-editor": "9.13.0",
-    "@lblod/ember-rdfa-editor-lblod-plugins": "19.3.1",
+    "@lblod/ember-mock-login": "^0.10.0",
+    "@lblod/ember-rdfa-editor": "9.14.0-dev.4448c75b33c1acd4715f63492d02980e7fb0662d",
```
Comment (lagartoverde) [inline package.json:49]: Wouldn't it be better to just set it to 9.14.0 and avoid these dev versions? or not sure what they are

**Review by lagartoverde — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (piemonkey) [reply package.json:49]: These are the development builds from the `release-commit` woodpecker pipeline for the editor or plugins. This is necessary to test with an unreleased version of those dependencies without requiring npm linking (or yalc). Given how much of a mess npm link and even yalc are, it's much more sane this way (especially when working on changing dependencies). We just need to make sure to change them for released versions before we release GN.

Comment (lagartoverde) [reply package.json:49]: So does this make sense to merge before those versions are released?

Comment (piemonkey) [reply package.json:49]: To me, yes, as it allows us to continue working on this (for example to fix the problems that Elena and I noticed, but want to do in a separate PR), without having to release many editor and plugins versions which may not be release ready as they need more testing. Happy to wait if others prefer though.

**Review by lagartoverde — APPROVED** [review]

**Comment by abeforgit** [comment]
I'm gonna merge this into #635 as the plugin release also contains breaking API changes along with the dependency changes, which are integrated in that pr

**Pull request merged**